test/e2e/restart.go

```diff
@@ -48,7 +48,9 @@ const (
 	restartPodReadyAgainTimeout = 5 * time.Minute
 )
 
-var _ = Describe("Restart", func() {
+// TODO(ihmccreery): This is skipped because it was previously in
```

Comment:
Not yet, but that's what my next task is, going through all the `[Skipped]` tests and filing issues for them.